PSKoans/Classes/Blank.ps1

```diff
@@ -0,0 +1,9 @@
+class Blank {
+    [string] ToString() {
+        return $null
+    }
+    
+    [bool] op_Equals([object] $other) {
+        return $false
+    }
+}
```

Comment:
Pretty sure this will need to be in the dummy types file so that it's externally visible.

Reply:
Doesn't seem like it, I just updated the output type to encapsulate as a string vs calling the object and that seems to have corrected the issue as its now passing the tests here.

Reply:
Interesting! Well, as long as it works, I'm happy with it. 😄